src/findings_data_import.py

```diff
@@ -49,12 +49,292 @@ def move_processed_file(s3_client, bucket, folder, filename):
+
+    field_map : str
+        The S3 key for the JSON file containing a map of incoming field names
+        to what they should be for the database.
+
+    Returns
+    -------
+    dict : The field map dict object
+    """
+    try:
+        # Log what/where up front so the subsequent messages make more sense
+        logging.info(
+            "Attempting to read Configuration data from %s in %s", field_map, s3_bucket
+        )
+        # Fetch object for the field_map JSON
+        field_map_object = s3_client.get_object(Bucket=s3_bucket, Key=field_map)
+        # Load field_map JSONs
+        field_map_dict = json.loads(
+            field_map_object.get("Body", "{}").read().decode("utf-8")
+        )
+        logging.info("Configuration data loaded from %s", field_map)
+        logging.debug("Configuration data: %s", field_map_dict)
+
+        return field_map_dict
+    except ClientError as client_err:
+        raise Exception(
+            f"Unable to download the field map data {field_map} from {s3_bucket}",
+            client_err,
+        )
+    except json.JSONDecodeError as json_err:
+        raise Exception(
+            "Unable to decode field map data, does not appear to be valid JSON.",
+            json_err,
+        )
+
+
+def setup_database_connection(
+    db_hostname=None,
+    db_port=None,
+    ssm_db_name=None,
+    ssm_db_user=None,
+    ssm_db_password=None,
+):
+    """Set up a mongo db connection based on the supplied host/port and SSM key values.
+
+    Parameters
+    ----------
+    db_hostname : str
+        The hostname that has the database to store the data in.
+
+    db_port : str
+        The port that the database server is listening on. [default: 27017]
+
+    ssm_db_name : str
+        The name of the parameter in AWS SSM that holds the name of the
+        database to store the assessment data in.
+
+    ssm_db_user : str
+        The name of the parameter in AWS SSM that holds the database username
+        with write permission to the assessment database.
+
+    ssm_db_password : str
+        The name of the parameter in AWS SSM that holds the database password
+        for the user with write permission to the assessment database.
+
+    Returns
+    -------
+    database : A database object returned from the MongoClient
+    """
+    try:
+        logging.info("Grabbing database credentials from SSM.")
+        # Fetch database credentials from AWS SSM
+        ssm_client = boto3_client("ssm")
+
+        db_info = dict()
+        for ssm_param_name, key in (
+            (ssm_db_name, "db_name"),
+            (ssm_db_user, "username"),
+            (ssm_db_password, "password"),
+        ):
+            response = ssm_client.get_parameter(
+                Name=ssm_param_name, WithDecryption=True
+            )
+            db_info[key] = response["Parameter"]["Value"]
+
+        logging.info("Connecting to the mongo db at %s %s", db_hostname, db_port)
```

Comment:
If you change `db_port` to an `int`, then this should change too:
```suggestion
        logging.info("Connecting to the mongo db at %s %d", db_hostname, db_port)
```